tests/actor_test.rs

```diff
@@ -1,379 +1,379 @@
-        0.into(),
-    );
-    h.expect_send(
-        &st,
-        &Address::new_id(ext::sca::SCA_ACTOR_ADDR),
-        Method::CommitChildCheckpoint as u64,
-        RawBytes::serialize(ch.clone()).unwrap(),
-        0.into(),
-    );
-    // 2/3 votes. Checkpoint committed
-    h.verify_checkpoint(&st, &epoch, Some(&ch));
-    // votes should have been cleaned
-    h.verify_check_votes(&st, &ch.cid(), 0);
-
-    // Trying to submit an already committed checkpoint should fail
-    let sender = h.senders.get_sender_by_index(2).unwrap();
-    h.submit_checkpoint(sender, epoch, &Cid::default(), ExitCode::USR_ILLEGAL_STATE);
-
-    // If the epoch is wrong in the next checkpoint, it should be rejected.
-    let prev_cid = ch.cid();
-    let sender = h.senders.get_sender_by_index(0).unwrap();
-    h.submit_checkpoint(sender, 11, &prev_cid, ExitCode::USR_ILLEGAL_STATE);
-
-    // Only validators should be entitled to submit checkpoints.
-    let epoch = 20;
-    let sender = h.senders.get_sender_by_index(3).unwrap();
-    h.submit_checkpoint(sender, epoch, &prev_cid, ExitCode::USR_ILLEGAL_STATE);
-
-    let sender = h.senders.get_sender_by_index(0).unwrap();
-    // Using wrong prev_cid should fail
-    h.submit_checkpoint(sender, epoch, &Cid::default(), ExitCode::USR_ILLEGAL_STATE);
-
-    // Submit checkpoint for subsequent epoch
-    let ch = h.submit_checkpoint(sender, epoch, &prev_cid, ExitCode::OK);
-    let st = h.get_state();
-    h.verify_check_votes(&st, &ch.cid(), 1);
-    h.expect_send(
-        &st,
-        &sender,
-        ext::account::PUBKEY_ADDRESS_METHOD,
-        RawBytes::default(),
-        0.into(),
-    );
-    // no checkpoint committed yet.
-    h.verify_checkpoint(&st, &epoch, None);
-
-    let sender = h.senders.get_sender_by_index(1).unwrap();
-    let ch = h.submit_checkpoint(sender, epoch, &prev_cid, ExitCode::OK);
-    let st = h.get_state();
-    h.expect_send(
-        &st,
-        &sender,
-        ext::account::PUBKEY_ADDRESS_METHOD,
-        RawBytes::default(),
-        0.into(),
-    );
-    h.expect_send(
-        &st,
-        &Address::new_id(ext::sca::SCA_ACTOR_ADDR),
-        Method::CommitChildCheckpoint as u64,
-        RawBytes::serialize(ch.clone()).unwrap(),
-        0.into(),
-    );
-    // 2/3 votes. Checkpoint committed
-    h.verify_checkpoint(&st, &epoch, Some(&ch));
-    // votes should have been cleaned
-    h.verify_check_votes(&st, &ch.cid(), 0);
-}
-
-fn std_params() -> ConstructParams {
-    ConstructParams {
-        parent: SubnetID::from_str("/root").unwrap(),
-        name: String::from("test"),
-        consensus: ConsensusType::PoW,
-        min_validator_stake: TokenAmount::from(10_u64.pow(18)),
-        min_validators: 1,
-        finality_threshold: 5,
-        check_period: 10,
-        genesis: Vec::new(),
-    }
-}
-
-fn std_join_params() -> JoinParams {
-    JoinParams {
-        validator_net_addr: String::from(":1234"),
-    }
-}
+// use std::str::FromStr;
```

Comment:
I would recommend adding a simple constructor test at least before merging to see that everything works (I think it may save us a lot of headache in the next PR)